site/docs/components/checkbox/accessibility.mdx

```diff
@@ -13,27 +13,27 @@ data:
 
 - If you are using a checkbox without a text description, pass an `aria-label` to the checkbox that describes its purpose.
 - On focus, the screen reader should:
```

Comment:
Same as before, I don't think screen reader content helps the end user

Reply:
Not too sure, I don't think anyone would indicate state like that and that would be broader labelling advice not specific to checkbox